media/js/base/global.js

```diff
@@ -81,6 +81,13 @@ function initLangSwitcher() {
     });
 }
 
+// client-side redirects (handy for testing)
+function doRedirect(destination) {
```

Comment:
This function helps us write unit tests against scripts that perform a redirect. Might be worth using this function in other places where we currently do `window.location.href = `, but that's another PR.